openc3-redis/docker-entrypoint.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+set -e
+
+# first arg is `-f` or `--some-option`
+# or first arg is `something.conf`
+if [ "${1#-}" != "$1" ] || [ "${1%.conf}" != "$1" ]; then
+	set -- redis-server "$@"
+fi
+
+um="$(umask)"
+if [ "$um" = '0022' ]; then
+	umask 0077
+fi
+
+exec "$@"
```

Comment:
What's the point of this and how are you accomplishing it?

Reply:
This is just a simplified version of the open source one to not change user ids during startup.  Exactly the same as ironbanks.